daemon/cluster/cluster.go

```diff
@@ -270,6 +270,16 @@ func (c *Cluster) GetAdvertiseAddress() string {
 	return c.currentNodeState().actualLocalAddr
 }
 
+// GetDataPathAddress returns the address to be used for the data path traffic.
+func (c *Cluster) GetDataPathAddress() string {
```

Comment:
Take it back, this is actually needed when libnetwork retrieves the addresss from the clusterProvider